### PR TITLE
*: add metrics when operator exceed store limit and the max cpu process

### PR DESCRIPTION
**Pull request by nolouch**
### What problem does this PR solve?

Issue Number: Close https://github.com/tikv/pd/issues/6043

### What is changed and how does it work?

```commit-message
*: add metrics when operator exceed store limit and the max cpu process
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
image

image
(default hide for noise reason.)
### Release note

```release-note
None.
```

### CODE REVIEW

**Review by bufferflies — APPROVED** [review]

**Comment by bufferflies** [comment]
UT failed

File: server/schedule/operator_controller.go [hunk 1]
```diff
@@ -847,6 +851,7 @@ func (oc *OperatorController) exceedStoreLimitLocked(ops ...*operator.Operator)
 				return false
 			}
 			if !limiter.Available(stepCost, v) {
+				operator.OperatorExceedStoreLimitCounter.WithLabelValues(desc).Inc()
```
Comment (lhy1024) [inline server/schedule/operator_controller.go:854]: only for urgent？

Comment (nolouch) [reply server/schedule/operator_controller.go]: urgent priority not limited by store limit.

**Review by lhy1024 — APPROVED** [review]

**Pull request merged**